src/features/leda-nft/store/leda-nft.slice.ts

```diff
@@ -42,6 +42,8 @@ const ledaNftSlice = createSlice({
 
 export const selectState = (state: RootState) => state.ledaNft;
 
+export const selectAuthData = (state: RootState) => state.auth;
```

Comment:
Please select a piece of a state from the corresponding slice. Here you are mixing two slices in one. (auth and ledaNft)